src/test/run-pass-fulldeps/create-dir-all-bare.rs

```diff
@@ -12,11 +12,11 @@
 
 #![feature(rustc_private)]
```

Comment:
I suppose you can remove this test if we're moving to `tempdir` from crates.io

Reply:
The reason as to why we don't change or remove the tests it's because the tests just happen to use `tempdir`, these don't test `tempdir` itself.